pkg/errors/helper.go

```diff
@@ -93,3 +94,33 @@ func IsRetryableError(err error) bool {
 	}
 	return true
 }
+
+// HTTPBadRequestError is some errors that will cause a BadRequestError in http handler
+var HTTPBadRequestError = []*errors.Error{
+	ErrAPIInvalidParam, ErrSinkURIInvalid, ErrStartTsBeforeGC,
+	ErrChangeFeedNotExists, ErrTargetTsBeforeStartTs, ErrTableIneligible,
+	ErrFilterRuleInvalid, ErrChangefeedUpdateRefused, ErrMySQLConnectionError,
+	ErrMySQLInvalidConfig,
+}
+
+// IsHTTPBadRequestError check if a error is a http bad request error
+func IsHTTPBadRequestError(err error) bool {
```

Comment:
Maybe it should be in an http related package?

Reply:
I don't think so, putting this code here will make the code more concise and easy to read.

Reply:
I understand that this package is an error related to the whole project, and your method is only related to http, so I think it may not be appropriate to put it here